xscen/aggregate.py

```diff
@@ -188,24 +188,26 @@ def compute_deltas(
     deltas = xr.Dataset(coords=other_hz.coords, attrs=other_hz.attrs)
     # Calculate deltas
     for vv in list(ds.data_vars):
-        if (isinstance(kind, dict) and kind[vv] == "+") or kind == "+":
-            _kind = "absolute"
-        elif (isinstance(kind, dict) and kind[vv] == "/") or kind == "/":
-            _kind = "relative"
-        else:
-            raise ValueError("Delta 'kind' not understood.")
-
         v_name = (
             vv
             if rename_variables is False
             else f"{vv}_delta_{reference_horizon.replace('-', '_')}"
         )
+
         with xr.set_options(keep_attrs=True):
-            if _kind == "absolute":
+            if (isinstance(kind, dict) and kind[vv] == "+") or kind == "+":
+                _kind = "absolute"
                 deltas[v_name] = other_hz[vv] - ref[vv]
-            else:
+            elif (isinstance(kind, dict) and kind[vv] == "/") or kind == "/":
+                _kind = "relative"
                 deltas[v_name] = other_hz[vv] / ref[vv]
                 deltas[v_name].attrs["units"] = ""
+            elif (isinstance(kind, dict) and kind[vv] == "%") or kind == "%":
+                _kind = "percentage"
+                deltas[v_name] = (other_hz[vv] - ref[vv]) / ref[vv]
```

Comment:
```suggestion
                deltas[v_name] = 100 * (other_hz[vv] - ref[vv]) / ref[vv]
```
Or am I mistaken? If this was to vary around 1, the correct cf units would be "1" I think.